### ISSUE TITLE
beep sound issue 

### ISSUE THREAD

**Issue by Seeder101** [hexchat/hexchat#1658]
Hi guys... 
recently I have been using hexchat on debian 8 (precompiled) from the debian repo 
and suddenly I have beep sound issue 
I usually get a beep sound when someone highlighted me or PMed me 
here is the fun part, when I'm actually in the tab of the one who PMing me I don't get any beep sound since I'm already reading his message which make sense I guess? 
now suddenly each message he send I got a beep sound which it's really annoying, isn't it? 
TingPing, in IRC channel told me this feature isn't available at the moment, which I find it quite weird XD its been working with me for quite some time and now its just vanished :p 

anyway...I checked the omit option from the alert preferences and now its good for me ....
sorry for my bad english if you didn't understand what I meant XD please let me explain more about my idea

**Comment by TingPing** [comment]
As I mentioned I'm pretty sure that was never a feature, but it would be nice to have.

**Comment by badon** [comment]
@Seeder101 Until this new feature arrives, you can use the option to "Omit alerts while the window is focused". That will eliminate all of the sounds. That probably works for almost everybody, but for people with huge numbers of channels open (like me), they may still need to be alerted if there is activity in one of them even when the window is focused.

**Comment by Seeder101** [comment]
@badon that's true and because of that since I have too many channels and maybe one or two queries that option "omit alerts while the window is focused" is useless and I think my idea is being here #1617 
before my opened issue here!
anyway, I can deal with it by the default settings even if it's annoying when you are actually chatting with someone and getting alerts with ever message =p 
I hope they will add the omit alerts when tab is focused because I don't want to don't being alerted by other channel or queries 
thank you again for stopping by
